src/Documentation/DocumentationGenerator/Extensions.cs

```diff
@@ -296,15 +296,15 @@ item.Item.VariableName switch
                     $"'{typeParam.Item.TypeName}",
                 _ => type.Resolution.Tag switch
                     {
-                        ResolvedTypeKind.Tags.BigInt => "[BigInt](xref:microsoft.quantum.lang-ref.bigint)",
-                        ResolvedTypeKind.Tags.Bool => "[Bool](xref:microsoft.quantum.lang-ref.bool)",
-                        ResolvedTypeKind.Tags.Double => "[Double](xref:microsoft.quantum.lang-ref.double)",
-                        ResolvedTypeKind.Tags.Int => "[Int](xref:microsoft.quantum.lang-ref.int)",
-                        ResolvedTypeKind.Tags.Pauli => "[Pauli](xref:microsoft.quantum.lang-ref.pauli)",
-                        ResolvedTypeKind.Tags.Qubit => "[Qubit](xref:microsoft.quantum.lang-ref.qubit)",
-                        ResolvedTypeKind.Tags.Range => "[Range](xref:microsoft.quantum.lang-ref.range)",
-                        ResolvedTypeKind.Tags.String => "[String](xref:microsoft.quantum.lang-ref.string)",
-                        ResolvedTypeKind.Tags.UnitType => "[Unit](xref:microsoft.quantum.lang-ref.unit)",
+                        ResolvedTypeKind.Tags.BigInt => "[BigInt](xref:microsoft.quantum.user-guide.language.types)",
+                        ResolvedTypeKind.Tags.Bool => "[Bool](xref:microsoft.quantum.user-guide.language.types)",
+                        ResolvedTypeKind.Tags.Double => "[Double](xref:microsoft.quantum.user-guide.language.types)",
+                        ResolvedTypeKind.Tags.Int => "[Int](xref:microsoft.quantum.user-guide.language.types)",
+                        ResolvedTypeKind.Tags.Pauli => "[Pauli](xref:microsoft.quantum.user-guide.language.types)",
+                        ResolvedTypeKind.Tags.Qubit => "[Qubit](xref:microsoft.quantum.concepts.the-qubit)",
+                        ResolvedTypeKind.Tags.Range => "[Range](xref:microsoft.quantum.user-guide.language.types)",
+                        ResolvedTypeKind.Tags.String => "[String](xref:microsoft.quantum.user-guide.language.types)",
+                        ResolvedTypeKind.Tags.UnitType => "[Unit](xref:microsoft.quantum.user-guide.language.types)",
```

Comment:
I believe that @geduardo just finished getting these links working, I'd suggest checking with first to see how the language reference / documentation integration is going. In any case, I'd suggest `microsoft.quantum.user-guide.language.types` over `microsoft.quantum.concepts.the-qubit` for `Qubit`.

Reply:
Good catch! I think the new uid for types is: `microsoft.quantum.qsharp.typesystem-index`.

This is the list of new uids: 

> 1_ProgramStructure/1_Namespaces.md,microsoft.quantum.qsharp.namespaces
1_ProgramStructure/2_TypeDeclarations.md,microsoft.quantum.qsharp.typedeclarations
1_ProgramStructure/3_CallableDeclarations.md,microsoft.quantum.qsharp.callabledeclarations
1_ProgramStructure/4_SpecializationDeclarations.md,microsoft.quantum.qsharp.specializationdeclarations
1_ProgramStructure/5_Attributes.md,microsoft.quantum.qsharp.attributes
1_ProgramStructure/6_AccessModifiers.md,microsoft.quantum.qsharp.accessmodifiers
1_ProgramStructure/7_Comments.md,microsoft.quantum.qsharp.comments
1_ProgramStructure/README.md,microsoft.quantum.qsharp.programstructure-index
2_Statements/VariableDeclarationsAndReassignments.md,microsoft.quantum.qsharp.variabledeclarationsandreassignments
2_Statements/ReturnsAndTermination.md,microsoft.quantum.qsharp.returnsandtermination
2_Statements/QuantumMemoryManagement.md,microsoft.quantum.qsharp.quantummemorymanagement
2_Statements/README.md,microsoft.quantum.qsharp.statements-index
2_Statements/Iterations.md,microsoft.quantum.qsharp.iterations
2_Statements/Conjugations.md,microsoft.quantum.qsharp.conjugations
2_Statements/ConditionalLoops.md,microsoft.quantum.qsharp.conditionalloops
2_Statements/ConditionalBranching.md,microsoft.quantum.qsharp.conditionalbranching
2_Statements/CallStatements.md,microsoft.quantum.qsharp.callstatements
2_Statements/BindingScopes.md,microsoft.quantum.qsharp.bindingscopes
3_Expressions/ValueLiterals.md,microsoft.quantum.qsharp.valueliterals
3_Expressions/PrecedenceAndAssociativity.md,microsoft.quantum.qsharp.precedenceandassociativity
3_Expressions/PartialApplication.md,microsoft.quantum.qsharp.partialapplication
3_Expressions/LogicalExpressions.md,microsoft.quantum.qsharp.logicalexpressions
3_Expressions/ItemAccessExpressions.md,microsoft.quantum.qsharp.itemaccessexpression
3_Expressions/README.md,microsoft.quantum.qsharp.expressions-index
3_Expressions/Identifiers.md,microsoft.quantum.qsharp.identifiers
3_Expressions/FunctorApplication.md,microsoft.quantum.qsharp.functorapplication
3_Expressions/CopyAndUpdateExpressions.md,microsoft.quantum.qsharp.copyandupdateexpressions
3_Expressions/ContextualExpressions.md,microsoft.quantum.qsharp.contextualexpressions
3_Expressions/ConditionalExpressions.md,microsoft.quantum.qsharp.conditionalexpressions
3_Expressions/Concatentation.md,microsoft.quantum.qsharp.concatenationexpressions
3_Expressions/ComparativeExpressions.md,microsoft.quantum.qsharp.comparativeexpressions
3_Expressions/BitwiseExpressions.md,microsoft.quantum.qsharp.bitwiseexpressions
3_Expressions/ArithmeticExpressions.md,microsoft.quantum.qsharp.arithmeticexpressions
4_TypeSystem/TypeParameterizations.md,microsoft.quantum.qsharp.typeparameterizations
4_TypeSystem/TypeInference.md,microsoft.quantum.qsharp.typeinference
4_TypeSystem/SubtypingAndVariance.md,microsoft.quantum.qsharp.subtypingandvariance
4_TypeSystem/SingletonTupleEquivalence.md,microsoft.quantum.qsharp.singletontupleequivalence
4_TypeSystem/QuantumDataTypes.md,microsoft.quantum.qsharp.quantumdatatypes
4_TypeSystem/OperationsAndFunctions.md,microsoft.quantum.qsharp.operationsandfunctions
4_TypeSystem/README.md,microsoft.quantum.qsharp.typesystem-index
4_TypeSystem/Immutability.md,microsoft.quantum.qsharp.immutability
5_Grammar/README.md,microsoft.quantum.qsharp.grammar